flink-dist/src/main/flink-bin/bin/pyflink-gateway-server.sh

```diff
@@ -50,8 +50,8 @@ done
 log=$FLINK_LOG_DIR/flink-$FLINK_IDENT_STRING-python-$HOSTNAME.log
 log_setting=(-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4j-cli.properties -Dlogback.configurationFile=file:"$FLINK_CONF_DIR"/logback.xml)
 
-TABLE_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-table*.jar`
-PYTHON_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-python*java-binding.jar`
+TABLE_JAR_PATH=`echo "$FLINK_HOME"/opt/flink-table*.jar`
+PYTHON_JAR_PATH=`echo "$FLINK_HOME"/opt/flink-python*java-binding.jar`
```

Comment:
🙄 this was committed after I wrote my change :/

Reply:
"Rebase translation" based on: https://github.com/pnowojski/flink/blob/FLINK-12143/flink-dist/src/main/flink-bin/bin/pyflink-gateway-server.sh#L53
:)